src/test/test_dataset_utils.py

```diff
@@ -19,3 +22,46 @@ def test_get_video_InBounds(self):
                 self.assertEqual(self.video_indexer.get_video_frame(absolute_idx),
                                  (video_idx, relative_frame_idx, frame_data))
                 absolute_idx += 1
+
+
+class TestDatasetUtilsFunctions(TestCase):
+    def test_project_to_image(self):
+        point_3d = np.array([[1., 2., 3.]])
+        calib = np.array([[1., 0., 2., 0.],
+                          [0., 3., 4., 0.],
+                          [0., 0., 1., 0.]])
+
+        point_2d = dataset_utils.project_to_image(point_3d, calib)
+        np.testing.assert_equal(point_2d, np.array([[7 / 3, 18 / 3]]))
+
+    def test_rot_y2alpha(self):
+        self.assertEqual(dataset_utils.rot_y2alpha(np.pi / 4, -400, 1200, 1600), np.pi / 2)
```

Comment:
Same here, you can use [`assertAlmostEqual`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual)